pom.xml

```diff
@@ -143,7 +128,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>${maven.surefire.version}</version>
+                <version>3.0.0-M4</version>
```

Comment:
Why did you remove the property?

Reply:
Because we use the version in only one place, so it went directly to its dependency.

Reply:
:+1:

Reply:
Maybe we should do this also if we use it in two places. I'm not sure if the versions plugin can update it otherwise.

Reply:
It can, it has `update-properties`